server/app/views/templates/mainFlashToUser.scala.html

```diff
@@ -3,6 +3,6 @@
 @(title: String)(content: Html)(implicit flash: Flash, msg: Messages, session: Session)
 
 @main(title) {
-    <div class="flashToUser">@flash.get(controllers.flashToUser).getOrElse("")</div>
+    @flash.get(controllers.flashToUser).map { f => <div class="flashToUser">@f</div> }
```

Comment:
did you check whether html view remains the same when div tag is not always printed?